plenum/test/node_request/message_request/test_node_requests_missing_preprepare.py

```diff
@@ -23,7 +23,8 @@ def test_node_requests_missing_preprepare(looper, txnPoolNodeSet,
     A node has bad network with primary and thus loses PRE-PREPARE,
     it requests PRE-PREPARE from non-primaries once it has sufficient PREPAREs
```

Comment:
The test description is wrong. It should be that `it requests PRE_PREPARE from the Primary`

Reply:
done